test/cmd/triggers.sh

```diff
@@ -72,16 +83,20 @@ os::test::junit::declare_suite_start "cmd/triggers/deploymentconfigs"
 # error conditions
 os::cmd::expect_failure_and_text 'oc set triggers dc/ruby-hello-world --from-github' 'deployment configs do not support GitHub web hooks'
 os::cmd::expect_failure_and_text 'oc set triggers dc/ruby-hello-world --from-webhook' 'deployment configs do not support web hooks'
+os::cmd::expect_failure_and_text 'oc set triggers dc/ruby-hello-world --from-gitlab' 'deployment configs do not support GitLab web hooks'
+os::cmd::expect_failure_and_text 'oc set triggers dc/ruby-hello-world --from-bitbucket' 'deployment configs do not support Bitbucket web hooks'
 os::cmd::expect_failure_and_text 'oc set triggers dc/ruby-hello-world --from-image=test:latest' 'you must specify --containers when setting --from-image'
 os::cmd::expect_failure_and_text 'oc set triggers dc/ruby-hello-world --from-image=test:latest --containers=other' 'not all container names exist: other \(accepts: ruby-hello-world\)'
 # print
 os::cmd::expect_success_and_text 'oc set triggers dc/ruby-hello-world' 'config.*true'
 os::cmd::expect_success_and_text 'oc set triggers dc/ruby-hello-world' 'image.*ruby-hello-world:latest \(ruby-hello-world\).*true'
 os::cmd::expect_success_and_not_text 'oc set triggers dc/ruby-hello-world' 'webhook'
 os::cmd::expect_success_and_not_text 'oc set triggers dc/ruby-hello-world' 'github'
+os::cmd::expect_success_and_not_text 'oc set triggers dc/ruby-hello-world' 'gitlab'
+os::cmd::expect_success_and_not_text 'oc set triggers dc/ruby-hello-world' 'bitbucket'
 # remove all
 os::cmd::expect_success_and_text 'oc set triggers dc/ruby-hello-world --remove-all' 'updated'
-os::cmd::expect_success_and_not_text 'oc set triggers dc/ruby-hello-world' 'webhook|github|image'
+os::cmd::expect_success_and_not_text 'oc set triggers dc/ruby-hello-world' 'webhook|github|image|gitlab|bitbucket'
```

Comment:
can reuse this pattern on lines 93-96 above, right?  not even sure we need both tests.